src/sonic-config-engine/minigraph.py

```diff
@@ -180,6 +180,7 @@ def parse_dpg(dpg, hname):
 
         pcintfs = child.find(str(QName(ns, "PortChannelInterfaces")))
         pc_intfs = []
```

Comment:
As PC interfaces are no longer in minigraph_interfaces (actually not in this PR but in #381), I think we need to update bgpd.conf.j2 as well to configure pc interfaces correctly?